app/client/components/user/CreateVendorForm.tsx

```diff
@@ -60,33 +44,33 @@ class CreateVendorForm extends React.Component<InjectedFormProps<ICreateUser, Pr
 
     if (!window.IS_USING_BITCOIN) {
       uniqueIdentifierInput = (
-        <Field name="publicSerialNumber" component={TextInput} type="text" label={isCashierAccountValue ? 'Cashier Phone Number' : 'Phone Number or ID'}>
+        <InputField name="publicSerialNumber" label={isCashierAccountValue ? 'Cashier Phone Number' : 'Phone Number or ID'} isRequired>
           <QrReadingModal
             updateData={ (data: string) =>  this.setSerialNumber(data) }
           />
-        </Field>
+        </InputField>
       )
     } else {
-      uniqueIdentifierInput = <Field name="blockchainAddress" component={TextInput} type="text" label='Bitcoin Address' />
+      uniqueIdentifierInput = <InputField name="blockchainAddress" label='Bitcoin Address' isRequired />
```

Comment:
we're actually deprecating support for bitcoin, so we could probably just kill these bits off.